### ISSUE TITLE
expand readthedocs

### ISSUE THREAD

**Issue by fsofras** [zeth/inputs#73]
It would be nice for many of us to have a list of methods and properties of this excellent library

**Comment by isibley765** [comment]
An API listing would be incredibly useful, yes

Listing classes, functions, and variables and how to reference them

Right now I can't get a GamePad class and figure out how to reference its logged events on its own without the while loop in your examples, and that while loop locking up processing for anything else isn't useful for more dynamic applications.

**Comment by Batcastle** [comment]
I agree. I need to be able to figure out when a button on a gamepad is being pressed cause the example just shows analogue stick coordinates when run.

**Comment by czimm79** [comment]
This is what I am trying to do! Did anyone figure out how to reference the logged events without a while loop? I'm currently using it for a PyQt application but even inside of a QThread it lags the rest of my application..
